example_notebooks/6_mls_wheat_demo.ipynb

```diff
@@ -0,0 +1,2666 @@
+{
```

Comment:
<p>Here, we should add something about the turn types of the platform, because they are interesting:</p><p>The ground vehicle tries to mimic real vehicles. It performs "smooth turns" for wide-angle curves. For narrow-angle curves, it first turns, then backs up, then finishes the turn (2 point turn). This can also be seen in the trajectory and we might also just give a short teaser here and add the detailed explanation below the figure.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/3dgeo-heidelberg/helios/pull/303/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>